### DISCUSSION TITLE
ImportError: cannot import name 'Github' from 'github' (/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/github/__init__.py)

### DISCUSSION THREAD

**Discussion by gitblanc** [PyGithub/PyGithub#2367] [Q&A]
I'm trying to list all the contents of a repository recursively but it seems like the import always crash. I don't really know why, because I've checked the route of installation at it's well installed. I'm trying this code:

````
from github import Github


def list_content_repo():
    g = Github()
    # github username
    username = g.get_user("gitblanc")
    # Obtain the repository
    repo = username.get_repo("Obsidian-Notes")
    print(repo.get_contents(""))

list_content_repo()

````

I also runned this command -> pip3 install PyGithub and doesnt' work. Hope you can help me :)

**Comment by EnricoMi** [comment]
I would start at getting `pip3 install PyGithub` to work.
